### PR TITLE
logo image stabilization

### LINKED ISSUE #533: Logo image missing from Navbar
**( Under GCI )**

**Describe the bug**
The CircuitVerse Logo on the Navigation Bar disappears when the browser window is resized to 1016*676 px

**To Reproduce**
1. Go to 'circuitverse.org'
2. Resize the browser window to 1016*676 px
3. See error

**Expected behavior**
The Logo div element should have fixed width irrespective of the current dimensions of the browser window. This can be done by fixing a min width value for the logo image div element.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by sourabhb77**
Fixes #533 

I added css for logo image so that it will never get hide.

After adding css:
Capture1

### CODE REVIEW

File: public/css/search.css [hunk 1]
```diff
@@ -69,3 +69,7 @@
     border: 0;
     padding-left: 0.8rem;
 }
+
+.navbar-brand img {
+max-width =max-content;
```
Comment (tachyons) [inline public/css/search.css]: make indentation proper here

Comment (sourabhb77) [reply public/css/search.css]: indentation is proper now

File: public/css/search.css [hunk 2]
```diff
@@ -69,3 +69,7 @@
     border: 0;
     padding-left: 0.8rem;
 }
+
+.navbar-brand img {
+    max-width : max-content;
```
Comment (tachyons) [inline public/css/search.css]: No space after width

Comment (sourabhb77) [reply public/css/search.css]: @tachyons please check changes

**Comment by tachyons** [comment]
Just noticed that you are making changes in old ui, can you please verify the issue and fix in the master branch? You can use gitpod in case you don't have local Circuitverse setup

**Pull request closed without merging**